html/pfappserver/root/static.alt/src/views/Nodes/_components/NodeView.vue

```diff
@@ -19,84 +19,172 @@
                   <b-badge variant="warning" v-else-if="node.ip4.end_time">{{ $t('Inactive since') }} {{ node.ip4.end_time }}</b-badge>
                   <b-badge variant="danger" v-else>{{ $t('Inactive') }}</b-badge>
               </pf-form-row>
-              <pf-form-row :label="$t('IPv6 Address')" v-if="node.ip6">
+              <pf-form-row class="text-nowrap" :label="$t('IPv6 Address')" v-if="node.ip6">
                 {{ node.ip6.ip }}
                   <b-badge variant="success" v-if="node.ip6.active">{{ $t('Since') }} {{ node.ip6.start_time }}</b-badge>
                   <b-badge variant="warning" v-else-if="node.ip6.end_time">{{ $t('Inactive since') }} {{ node.ip6.end_time }}</b-badge>
                   <b-badge variant="danger" v-else>{{ $t('Inactive') }}</b-badge>
               </pf-form-row>
+              <pf-form-row class="text-nowrap" :label="$t('Detect Date')">
+                {{ node.detect_date }}
+              </pf-form-row>
+              <pf-form-row class="text-nowrap" :label="$t('Registration Date')">
+                {{ node.regdate }}
+              </pf-form-row>
+              <pf-form-row class="text-nowrap" :label="$t('Unregistration Date')">
+                {{ node.unregdate }}
+              </pf-form-row>
+              <pf-form-row class="text-nowrap" :label="$t('Last ARP')">
+                {{ node.last_arp }}
+              </pf-form-row>
+              <pf-form-row class="text-nowrap" :label="$t('Last DHCP')">
+                {{ node.last_dhcp }}
+              </pf-form-row>
+              <pf-form-row class="text-nowrap" :label="$t('Last Seen')">
+                {{ node.last_seen }}
+              </pf-form-row>
+              <pf-form-row class="text-nowrap" :label="$t('Last Skip')">
+                {{ node.lastskip }}
+              </pf-form-row>
+              <pf-form-row class="text-nowrap" :label="$t('Last Connection Type')">
+                {{ node.last_connection_type }}
+              </pf-form-row>
+              <pf-form-row class="text-nowrap" :label="$t('Last Connection Sub Type')">
+                {{ node.last_connection_sub_type }}
+              </pf-form-row>
+              <pf-form-row class="text-nowrap" :label="$t('Last .1X Username')">
+                {{ node.last_dot1x_username }}
+              </pf-form-row>
+              <pf-form-row class="text-nowrap" :label="$t('Last Port')">
+                {{ node.last_port }}
+              </pf-form-row>
+              <pf-form-row class="text-nowrap" :label="$t('Last SSID')">
+                {{ node.last_ssid }}
+              </pf-form-row>
+              <pf-form-row class="text-nowrap" :label="$t('Last Start Time')">
+                {{ node.last_start_time }}
+              </pf-form-row>
+              <pf-form-row class="text-nowrap" :label="$t('Last Start Timestamp')">
+                {{ node.last_start_timestamp }}
+              </pf-form-row>
```

Comment:
Is the *start time* and *start timestamp* redundant information?

Reply:
Not sure, I haven't seen any live example with these fields populated.